src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -214,4 +357,155 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
         );
         return map;
     }
+
+    @autobind
+    protected mutationsGroupedByProteinChangeForGroup(groupIndex: number) {
+        let allGroupedData = this.store.dataStore.allData;
```

Comment:
try to avoid using ```let``` in general.  here, you could just do 
```const allGroupedData = groupDataByGroupFilters``` with third arg being this.store.data.store.allData, no?